Thank you ChatGPT.

In the first terminal, I ran: nickspi5@raspberrypi1:~ $ cd Chatty_AI
nickspi5@raspberrypi1:~/Chatty_AI $ source ~/Chatty_AI/chatty-venv/bin/activate
(chatty-venv) nickspi5@raspberrypi1:~/Chatty_AI $ python3 -m wyoming_faster_whisper \
  --uri tcp://0.0.0.0:10300 \
  --model tiny \
  --language en \
  --data-dir ~/Chatty_AI/whisper-models \
  --device cpu \
  --compute-type int8
INFO:__main__:Ready

In the second terminal I ran: (chatty-venv) nickspi5@raspberrypi1:~/Chatty_AI $ python3 -m wyoming_piper \
  --piper ~/Chatty_AI/piper/piper \
  --data-dir ~/Chatty_AI/voices \
  --voice en_US-amy-low \
  --uri tcp://0.0.0.0:10200


Is this an error we need to fix? If so, how do we fix it?

